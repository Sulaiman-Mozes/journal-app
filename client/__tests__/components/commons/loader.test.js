import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import Loader from '../../../components/commons/loader';

describe('Provider and Loader', () => {
  const loader = shallow(<Loader />);

  it('renders <Provider/> correctly', () => {
    expect(loader).toMatchSnapshot();
  });
});
