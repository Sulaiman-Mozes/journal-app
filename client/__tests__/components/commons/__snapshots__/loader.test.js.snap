// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Provider and Loader renders <Provider/> correctly 1`] = `
<div
  className="flex-notfound text-center"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
